frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -85,7 +85,12 @@ describe('Kubernetes resource CRUD operations', () => {
       }
 
       it('displays a YAML editor for creating a new resource instance', async() => {
-        await crudView.createYAMLButton.click();
+        if (kind === 'Secret') {
```

Comment:
Instead of checking `kind`, can we check for the existence of `#item-create`? Then if we add similar buttons to other pages, the tests don't need to change.